db/views/reporting_patient_states_v06.sql

```diff
@@ -0,0 +1,217 @@
+    cal.month,
+    cal.quarter,
+    cal.year,
+    cal.month_string,
+    cal.quarter_string,
+
+    ------------------------------------------------------------
+    -- medical history
+    mh.hypertension as hypertension,
+    mh.prior_heart_attack as prior_heart_attack,
+    mh.prior_stroke as prior_stroke,
+    mh.chronic_kidney_disease as chronic_kidney_disease,
+    mh.receiving_treatment_for_hypertension as receiving_treatment_for_hypertension,
+    mh.diabetes as diabetes,
+
+    ------------------------------------------------------------
+    -- information on assigned facility and parent regions
+    p.assigned_facility_id AS assigned_facility_id,
+    assigned_facility.facility_size as assigned_facility_size,
+    assigned_facility.facility_type as assigned_facility_type,
+    assigned_facility.facility_region_slug as assigned_facility_slug,
+    assigned_facility.facility_region_id as assigned_facility_region_id,
+    assigned_facility.block_slug as assigned_block_slug,
+    assigned_facility.block_region_id as assigned_block_region_id,
+    assigned_facility.district_slug as assigned_district_slug,
+    assigned_facility.district_region_id as assigned_district_region_id,
+    assigned_facility.state_slug as assigned_state_slug,
+    assigned_facility.state_region_id as assigned_state_region_id,
+    assigned_facility.organization_slug as assigned_organization_slug,
+    assigned_facility.organization_region_id as assigned_organization_region_id,
+
+    ------------------------------------------------------------
+    -- information on registration facility and parent regions
+    p.registration_facility_id AS registration_facility_id,
+    registration_facility.facility_size as registration_facility_size,
+    registration_facility.facility_type as registration_facility_type,
+    registration_facility.facility_region_slug as registration_facility_slug,
+    registration_facility.facility_region_id as registration_facility_region_id,
+    registration_facility.block_slug as registration_block_slug,
+    registration_facility.block_region_id as registration_block_region_id,
+    registration_facility.district_slug as registration_district_slug,
+    registration_facility.district_region_id as registration_district_region_id,
+    registration_facility.state_slug as registration_state_slug,
+    registration_facility.state_region_id as registration_state_region_id,
+    registration_facility.organization_slug as registration_organization_slug,
+    registration_facility.organization_region_id as registration_organization_region_id,
+
+    ------------------------------------------------------------
+    -- details of the visit: latest BP, BS, encounter, prescription drug and appointment
+    bps.blood_pressure_id as blood_pressure_id,
+    bps.blood_pressure_facility_id AS bp_facility_id,
+    bps.blood_pressure_recorded_at AS bp_recorded_at,
+    bps.systolic,
+    bps.diastolic,
+
+    bss.blood_sugar_id as blood_sugar_id,
```

Comment:
We are adding the latest blood sugar measurement details as of a given month here.